### ISSUE TITLE
A mode to mimic Legion Go's mapping type "instant" for gyro

### ISSUE THREAD

**Issue by ivanbulanov** [Valkirie/HandheldCompanion#1182]
**Is your feature request related to a problem? Please describe.**
I am playing Mad Max. There is driving and typical 3rd person action. The game has a good controller support. It doesn't make sense to play it with mouse and keyboard emulation. However, it would benefit from gyro camera control.

**Describe the solution you'd like**
Legion Go Space has an option to emulate left or right stick movement with gyro. There are two mapping types: "continuous" and "instant". "Continuous" is what Handheld Companion does. It constantly translates gyro angle to stick value. "Instant" is more interesting. In instant mode the stick is always moving towards the central neutral position. When a user tilts the device, the stick temporarily shifts from the middle position and then returns to it again. This stick movement is combined with the physical stick movement. This approach makes it easy to do fine adjustments with gyro while retaining the ability to do fast and coarse adjustments with the physical stick.

**Describe alternatives you've considered**
The alternative is to turn off HC and enable Legion Space when playing Mad Max. I didn't consider emulating mouse and keyboard because I think it would be less convenient.

**Additional context**
![Screenshot of the option in Legion Space](https://github.com/user-attachments/assets/2297325e-f730-4480-ba8d-215d18beea4e)

**Comment by CasperH2O** [comment]
I think you are looking for local space and steering motion option type in HC. Switching between the two will have to be done through QT though.

https://m.youtube.com/watch?v=uTpIWsacB_M

**Comment by ivanbulanov** [comment]
@CasperH2O No, not really. Your example is more like the continuous mode. I'm looking for a mode that centers the stick if the device is stationary even when tilted. I haven't seen a good demo of it. This one is as good as I could find: https://www.youtube.com/watch?v=sAzIngkbmIU
